Sample.txt.txt

```diff
@@ -1,7 +1,4 @@
 1
 2
 3
-4
-5
-6
-7
\ No newline at end of file
+4
```

Comment:
- [ ] Please add back the line
- [ ] Please add some character instead on number